etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,375 @@
+    port: 11211
+    proto: tcp
+    destination: "{{ internal_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for memcached"
+  - interface: "{{ internal_net_interface }}"
+    port: 15672
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'beam.smp' in docker container 'rabbitmq'"
+  - interface: "{{ internal_net_interface }}"
+    port: 15672
+    proto: tcp
+    destination: "{{ internal_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for rabbitmq_management"
+  - interface: "{{ internal_net_interface }}"
+    port: 25672
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'beam.smp' in docker container 'rabbitmq'"
+  - interface: "{{ internal_net_interface }}"
+    port: 35357
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'httpd' in docker container 'keystone'"
+  - interface: "{{ internal_net_interface }}"
+    port: 35357
+    proto: tcp
+    destination: "{{ internal_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for keystone_admin"
+  - interface: "{{ public_net_interface }}"
+    port: 80
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for horizon_external"
+  - interface: "{{ public_net_interface }}"
+    port: 3000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for grafana_server_external"
+  - interface: "{{ public_net_interface }}"
+    port: 5000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for keystone_external"
+  - interface: "{{ public_net_interface }}"
+    port: 5601
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for kibana_external"
+  - interface: "{{ public_net_interface }}"
+    port: 6080
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_novncproxy_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_cfn_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8004
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8774
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8775
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_metadata_external"
```

Comment:
Not sure we want this one... although I need to check how the neutron proxy works.